Frontend/src/app/views/dashboard/time/time.component.scss

```diff
@@ -1,8 +1,7 @@
+@import "../../../../app/styles/display.scss";
+
 .container {
   padding: 20px;
-  padding-left: 10px;
-  padding-right: 10px;
-  overflow: hidden;
 }

```

Comment:
Can probably just remove this and use the padding that the shared `.container` gives

Reply:
Padding removed